Modules/Products/Tests/Feature/FamilyDeletionValidationFeatureTest.php

```
@@ -0,0 +1,136 @@
+    {
+        /** Arrange */
+        $family = Family::factory()->create(['family_name' => 'Empty Family']);
+
+        /** Act */
+        $response = $this->post(route('families.delete', ['id' => $family->family_id]));
+
+        /** Assert */
+        $response->assertRedirect(route('families.index'));
+        $response->assertSessionHas('alert_success');
+        $this->assertDatabaseMissing('ip_families', ['family_id' => $family->family_id]);
+    }
```

PHPMD (2.15.0)

The method it_deletes_family_without_products is not named in camelCase. (undefined)

(CamelCaseMethodName)